static/app/views/ddm/querySymbol.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from '@emotion/styled';
+
+const indexToChar = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+export const getQuerySymbol = (index: number) => {
+  let result = '';
+  let i = index;
+  do {
+    result = indexToChar[i % indexToChar.length] + result;
+    i = Math.floor(i / indexToChar.length) - 1;
+  } while (i >= 0);
+  return result;
+};
+
+const Symbol = styled('div')`
```

Comment:
nit: The letters feel uncentered.

![CleanShot 2024-01-10 at 17 47 34](https://github.com/getsentry/sentry/assets/9060071/55f0f8f6-3f21-4b2c-9380-6ba24b310d65)

Reply:
we get sub-pixel rendering issues here. looks centered on retina screens 🤔